.env.example

```
@@ -1,2 +1,4 @@
 BASE_PATH=
 IFRAME_TARGET_ORIGIN=
+AI_ASSISTANT_ENABLED=true
+AI_ASSISTANT_BASE_URL_REGEX="^(?:ki-toolbox\.scc\.kit\.edu|(?!(?:[a-z0-9-]+\.)*kit\.edu$)(?:[a-z0-9-]+\.)*[a-z0-9-]+)$"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_ASSISTANT_BASE_URL_REGEX key should go before the AI_ASSISTANT_ENABLED key

(UnorderedKey)